include/sys/time.h

```diff
@@ -5,13 +5,21 @@
 
 #define CLK_TCK 100 /* system clock ticks per second */
 
+/* 1ns = (2^64) / 1000000000 */
+#define BINTIME_SCALE_NS ((uint64_t)18446744073ULL)
```

Comment:
I don't like the idea of polluting public namespace with macrodefinitions that are used to implement a single procedure used by the kernel.